From 935269a7349f7b3aeaf8c949866b95f80e169d37 Mon Sep 17 00:00:00 2001
From: Valentin Caron <valentin.caron@foss.st.com>
Date: Thu, 5 Dec 2024 16:04:37 +0100
Subject: [PATCH 1405/1695] dts: stm32: add st,proc-id to ipcc node in
 stm32mp211.dtsi

Add this property in ippc1 (0: CA35 is CPU1) node

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git 2a1585c1235e3c89b9330b5a4d5019b1b9e8855d dts: stm32: add st,proc-id to ipcc node in stm32mp211.dtsi]
Signed-off-by: Valentin Caron <valentin.caron@foss.st.com>
Change-Id: Iddcd5be8c04cb6cfd6013772445162400513f35f
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/423181
Reviewed-by: Arnaud POULIQUEN <arnaud.pouliquen@st.com>
Reviewed-by: Thomas BOURGOIN <thomas.bourgoin@foss.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: Gwenael TREUVEUR <gwenael.treuveur@st.com>
Domain-Review: Thomas BOURGOIN <thomas.bourgoin@foss.st.com>
Reviewed-by: Michel JAOUEN <michel.jaouen@st.com>
---
 core/arch/arm/dts/stm32mp211.dtsi | 1 +
 1 file changed, 1 insertion(+)

diff --git a/core/arch/arm/dts/stm32mp211.dtsi b/core/arch/arm/dts/stm32mp211.dtsi
index 4b4dd5629..ae4691242 100644
--- a/core/arch/arm/dts/stm32mp211.dtsi
+++ b/core/arch/arm/dts/stm32mp211.dtsi
@@ -1982,6 +1982,7 @@
 			interrupts = <GIC_SPI 154 IRQ_TYPE_LEVEL_HIGH>;
 			interrupt-names = "rx";
 			clocks = <&rcc CK_BUS_IPCC1>;
+			st,proc-id = <0>;
 		};
 
 		rifsc: bus@42080000 {
-- 
2.47.2

